src/kernel/vmm.zig

```diff
@@ -777,7 +782,7 @@ test "copy" {
     std.testing.expectError(VmmError.NotAllocated, mirrored.virtToPhys(alloc4));
 }
 
-test "copyDataToVMM" {
+test "copyData" {
```

Comment:
Probs worth added some test for copying the other way around.

Reply:
That is actually below this. Have a look at the lines starting with `try vmm2.copyData(&vmm, buff2, alloc, false);`.

Reply:
Are yes, I am blind